.github/workflows/browser-tests.yaml

```diff
@@ -22,15 +22,39 @@ jobs:
       with:
         python-version: ${{ matrix.python-version }}
 
+    - name: Cache pip
+      uses: actions/cache@v2
+      with:
+        # This path is specific to Ubuntu
+        path: ~/.cache/pip
+        # Look to see if there is a cache hit for the corresponding requirements file
+        key: ${{ runner.os }}-pip-${{ hashFiles('requirements.txt') }}
+        restore-keys: |
+          ${{ runner.os }}-pip-
+          ${{ runner.os }}-
```

Comment:
Why are there two keys here? I don't see any issue with it, just curious. I saw this [question](https://github.community/t/pip-dependencies-not-getting-cached-even-after-successful-restore/126672) using the same keys in an answer, but it doesn't explain why this works.

Reply:
I don't have a great answer for you; it was featured that way in the examples.